src/fsharp/ErrorResolutionHints.fs

```diff
@@ -53,7 +53,7 @@ let FilterPredictions (idText:string) (suggestionF:ErrorLogger.Suggestions) =
         let suggestion:string = demangle suggestion
         let suggestedText = suggestion.ToUpperInvariant()
         let similarity = EditDistance.JaroWinklerDistance uppercaseText suggestedText
-        if similarity >= highConfidenceThreshold || suggestion.EndsWithOrdinal("." + idText) then
+        if similarity >= highConfidenceThreshold || (suggestion.[suggestion.Length - idText.Length - 1] = '.' && suggestion.EndsWithOrdinal(idText)) then
```

Comment:
Do you think that's faster? That one allocation probably doesn't hurt

Reply:
Probably a micro optimization. `FilterPredictions` can be called a fair bit over a long period of time, though I agree that the one allocation per call isn't severe